src/stylesheets/elements/_list.scss

```diff
@@ -1,50 +1,39 @@
-ul, ol {
```

Comment:
I'll call your attention to this file, where I did actually move things around to try to consolidate rules. If you don't mind, please double check to make sure I didn't miss anything :)